sql/moz-fx-data-shared-prod/telemetry/active_users_aggregates/view.sql

```diff
@@ -3,6 +3,43 @@ CREATE OR REPLACE VIEW
 AS
 SELECT
   *,
-  `mozfun.norm.os`(os) AS os_grouped
+  dau AS qdau
 FROM
-  `moz-fx-data-shared-prod.telemetry_derived.active_users_aggregates_v1`
+  `telemetry.active_users_aggregates_mobile`
+UNION ALL
+SELECT
+  segment,
+  attribution_medium,
+  attribution_source,
+  attributed,
+  city,
+  country,
+  distribution_id,
+  first_seen_year,
+  is_default_browser,
+  app_name,
+  channel,
+  os,
+  os_version,
+  os_version_major,
+  os_version_minor,
+  submission_date,
+  language_name,
+  dau,
```

Comment:
Not blocking or required, but a follow-up suggestion from the comment above.
```suggestion
  qdau AS dau,
```